hls4ml/convert.py

```diff
@@ -181,8 +192,34 @@ def main(args):
             plt.ylabel('QKeras {}'.format(klayer))
             plt.savefig(os.path.join(cfg['OutputDir'], 'profiling_{}.png'.format(layer)), dpi=300)
     else:
+        # Original save of input_data/output_preds
+        np.savetxt('input_data.dat', X_test[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+        np.savetxt('output_predictions.dat', y_test[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+    
+        # Extending input_data/output_pred by appending
+        # the data/preds of more test images (num_test_imgs)
+        for i in range(1, num_test_imgs):
+            with open('input_data.dat', "ab") as f:
+                np.savetxt(f, X_test[i].reshape(1, -1), fmt='%f', delimiter=' ')
+    
+            with open('output_predictions.dat', "ab") as f:
+                np.savetxt(f, y_test[i].reshape(1, -1), fmt='%f', delimiter=' ')
         hls_model.compile()
         y_hls = hls_model.predict(X_test)
+        ######################
+        # Original save of input_data/output_preds
+        np.savetxt('hls_predictions.dat', y_hls[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+        np.savetxt('keras_predictions.dat', y_keras[:1].reshape(1, -1), fmt='%f', delimiter=' ')
+        
+        # Extending input_data/output_pred by appending
+        # the data/preds of more test images (num_test_imgs)
+        for i in range(1, num_test_imgs):        
+          with open('hls_predictions.dat', "ab") as f:
+            np.savetxt(f, y_hls[i].reshape(1, -1), fmt='%f', delimiter=' ')
+            
+          with open('keras_predictions.dat', "ab") as f:
+            np.savetxt(f, y_keras[i].reshape(1, -1), fmt='%f', delimiter=' ')
+        #######################
```

Comment:
i could see wanting to add the keras/hls predicitions, but this could probably be done more compactly (i think).

```python
np.savetxt('hls_predictions.dat', y_hls[:num_test_imgs].reshape(num_test_imgs, -1), fmt='%f', delimiter=' ')
np.savetxt('keras_predictions.dat', y_keras[:num_test_imgs].reshape(num_test_imgs, -1), fmt='%f', delimiter=' ')
```
also `num_test_imgs` should be consistent with the size of `X_test` and `y_test`, which is currently 10 when we're building:

https://github.com/hls4ml-finn-mlperftiny/CIFAR10/blob/04ec51260e0931a9aca8cc283f4f793ce6792a6a/hls4ml/convert.py#L89-L91